### PR TITLE
[SD-3914] set fetch operation for fetch from AAP multimedia, do not p…

### PR DESCRIPTION
**Pull request by marwoodandrew**
…ass empty dateline from client

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -2156,9 +2156,8 @@
                                 scope.datelineDay = monthAndDay.day;
                                 scope.resetNumberOfDays(false);
                             }
+                            _.extend(item, updates);
```
Comment (mugurrus) [inline client/app/scripts/superdesk-authoring/authoring.js]: `var updates = {dateline: {}};` can be moved inside the inner if and then the 2 ifs can be combine in a single one.

**Comment by mugurrus** [comment]
:+1: (except comment)

File: server/apps/search_providers/aap_mm/service.py [hunk 2]
```diff
@@ -63,6 +64,7 @@ def create(self, docs, **kwargs):
             dest_doc[ITEM_STATE] = doc.get(ITEM_STATE, CONTENT_STATE.FETCHED)
             dest_doc[INGEST_ID] = archived_doc[config.ID_FIELD]
             dest_doc[FAMILY_ID] = archived_doc[config.ID_FIELD]
+            dest_doc[ITEM_OPERATION] = ITEM_FETCH
```
Comment (mdhaman) [inline server/apps/search_providers/aap_mm/service.py]: Do we have a behave test verifying that multimedia content fetch does not have dateline?

**Comment by akintolga** [comment]
:+1:

**Pull request merged**